Sources/Benchmark/BenchmarkArguments.swift

```diff
@@ -45,6 +45,9 @@ public struct BenchmarkArguments: ParsableArguments {
     @Option(help: "Comma-separated list of column names to show.")
     var columns: String?
 
+    @Option(help: "Output format.")
```

Comment:
```suggestion
    @Option(help: "Output format (valid values are: json, csv, console, none).")
```

Or something like that.